x/mongo/driver/auth/plain_test.go

```diff
@@ -26,6 +25,7 @@ func TestPlainAuthenticator_Fails(t *testing.T) {
 	authenticator := PlainAuthenticator{
 		Username: "user",
 		Password: "pencil",
+		Source:   "$external",
```

Comment:
We use this so much, what are your thoughts on constantizing?

Reply:
The tradeoff is more indirection for lower typo risk. Since the source of truth is the server APIs, there is no risk of out-of-sync constants in the Go Driver. Seems like a good idea overall, will update 👍